test/Tests/src/Semantic/Error_Spec.enso

```diff
@@ -413,7 +422,29 @@ spec =
             r . should_fail_with Type_Error
             r.to_display_text . should_contain "Try to apply a, b arguments"
 
-
+    Test.group "Wrapped errors" <|
+        Test.specify ".catch unwraps an error when the inner error is explicitly specified" <|
+            Error.throw (My_Error_Wrapper.Error (My_Error.Error 12)) . catch My_Error x->
+                x . should_equal (My_Error.Error 12)
```

Comment:
A bit of a nitpick, but I don't like tests that have a `should_equal` inside of constructs like `catch`.

The problem is - what if by some reason `catch` does not run the caught-block at all? Ofc most likely in that particular scenario that would mean that the error is forwarded and most likely the whole test will fail with 'unexpected dataflow error'.

But still it just seems safer to do sth like:
```suggestion
            r = Error.throw (My_Error_Wrapper.Error (My_Error.Error 12)) . catch My_Error x->x
            r.should_equal (My_Error.Error 12)
```